README.md

```diff
@@ -5,6 +5,13 @@
   - Docker
   - Java
   - Node
+  - application.yml: add this file to the ./resources folder
```

Comment:
Might be good to use the full path, `src/main/resources` for anyone not familiar with the structure